### PR TITLE
Quo2 Wallet: Summary Info

### LINKED ISSUE #16756: Quo2 Wallet: Summary Info component
Implement this component in the quo2 component library as per the design spec.

Reuse Quo2 components wherever applicable.

Designs
https://www.figma.com/file/WQZcp6S0EnzxdTL4taoKDv/Design-System-for-Mobile?type=design&node-id=22669-206023&mode=design&t=Yui6CQ6Hh2CplG7a-0

Acceptance Criteria:

component spec tests - https://github.com/status-im/status-mobile/blob/develop/doc/component-tests-overview.md
quo2 preview screen
export from quo2/core.cljs
following best practices - https://github.com/status-im/status-mobile/blob/develop/doc/new-guidelines.md

Note
Please align with design team before starting this

### PR DESCRIPTION
**Pull request by OmarBasem**
fixes: https://github.com/status-im/status-mobile/issues/16756

This PR implements Summary Info component.

[Designs.](https://www.figma.com/file/WQZcp6S0EnzxdTL4taoKDv/Design-System-for-Mobile?type=design&node-id=22669-206023&mode=design&t=Yui6CQ6Hh2CplG7a-0)

Some previews:

image_6487327 (2)
image_6487327
image_6487327 (1)

### CODE REVIEW

File: src/quo2/components/wallet/summary_info/view.cljs [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+       :divider? true
+       :theme    theme}]
+     [network-amount
+      {:network :arbitrum
+       :amount  (str arbitrum " ETH")
+       :theme   theme}]]))
+
+(defn- view-internal
+  [{:keys [theme type account-props networks? values]}]
+  (let []
```
Comment (rasom) [inline src/quo2/components/wallet/summary_info/view.cljs]: unnecessary `let`

**Review by rasom — APPROVED** [review]

File: src/quo2/components/wallet/summary_info/view.cljs [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+    {:size   :paragraph-2
+     :weight :medium
+     :style  {:margin-left 4}} amount]
+   (when divider?
+     [rn/view
+      {:style {:width             2
+               :height            2
+               :border-radius     2
+               :margin-horizontal 8
+               :background-color  (colors/theme-colors colors/neutral-40 colors/neutral-50 theme)}}])])
```
Comment (briansztamfater) [inline src/quo2/components/wallet/summary_info/view.cljs]: This should be in styles file

File: src/quo2/components/wallet/summary_info/view.cljs [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+
+(defn networks
+  [values theme]
+  (let [{:keys [ethereum optimism arbitrum]} values]
+    [rn/view
+     {:style               {:padding-horizontal 12
+                            :padding-vertical   4
+                            :flex-direction     :row
+                            :align-items        :center}
```
Comment (briansztamfater) [inline src/quo2/components/wallet/summary_info/view.cljs]: This should be in styles file

File: src/quo2/components/wallet/summary_info/view.cljs [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+    [quo2.components.avatars.user-avatar.view :as user-avatar]
+    [quo2.components.markdown.text :as text]
+    [quo2.foundations.colors :as colors]
+    [quo2.foundations.resources :as resources]
+    [quo2.theme :as quo.theme]
+    [quo2.components.avatars.account-avatar.view :as account-avatar]
+    [react-native.core :as rn]
+    [quo2.components.wallet.summary-info.style :as style]))
+
+
```
Comment (briansztamfater) [inline src/quo2/components/wallet/summary_info/view.cljs]: unnecessary empty line

File: src/quo2/components/wallet/summary_info/component_spec.cljs [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+               {:type          :status-account
+                :networks?     false
+                :values        {:ethereum 150
+                                :optimism 50
+                                :arbitrum 25}
+                :account-props status-account-props}])
+    (h/is-falsy (h/get-by-label-text :networks))))
+
+
+
```
Comment (briansztamfater) [inline src/quo2/components/wallet/summary_info/component_spec.cljs]: some unnecessary extra lines

**Review by briansztamfater — APPROVED** [review]
LGTM, just left some minor comments

**Comment by briansztamfater** [comment]
@OmarBasem One thing I noticed based on screenshots and comparing to Figma designs, is that the content in the bottom view is not vertically centered.

258370079-a55345bc-7768-44ba-b5d4-60f73dd347fd

**Comment by OmarBasem** [comment]
> @OmarBasem One thing I noticed based on screenshots and comparing to Figma designs, is that the content in the bottom view is not vertically centered.
> 

Fixed it! Thanks @briansztamfater

**Comment by OmarBasem** [comment]
Hi @Francesca-G, can I get a design review please?

**Review by Francesca-G — APPROVED** [review]
Here's the [design review ](https://www.figma.com/file/Tf5nfkYvpbnNCo4rKLK7lS/Feedback-for-Mobile?type=design&node-id=3446%3A227419&mode=design&t=SD5YCCfJwWLwww3h-1) :)

**Comment by OmarBasem** [comment]
Hi @Francesca-G, I have replied to your comments on Figma. Please let me know if a follow up is still required. Thanks!

**Review by Francesca-G — APPROVED** [review]

**Pull request merged**